python/lsst/obs/monocam/monocam.py

```diff
@@ -162,68 +181,64 @@ def _makeAmpInfoCatalog(self):
         linearityMax = saturation
         linearityCoeffs = [linearityThreshold, linearityMax]
 
-        schema = AmpInfoTable.makeMinimalSchema()
-
-        linThreshKey = schema.addField('linearityThreshold', type=float)
-        linMaxKey = schema.addField('linearityMaximum', type=float)
-        linUnitsKey = schema.addField('linearityUnits', type=str, size=9)
-        # end placeholder
-        self.ampInfoDict = {}
-        ampCatalog = AmpInfoCatalog(schema)
+        ampCatalog = []
         for ampY in (0, 1):
             for ampX in range(8):
-                record = ampCatalog.addNew()
-                record.setName("%d%d" % (ampX, ampY))
+                amplifier = cameraGeom.Amplifier.Builder()
```

Comment:
This may be outside of what I should be reviewing, but seeing this line I would expect that this would be creating a builder that will later produce an amplifier, and not an amp directly. Does this seems right? I see that there is no "build" called at the end of adding all the info, so I assume this is itself an amplifier.

Reply:
This isn't completely obvious, and is probably obscured somewhat by my object names.  `amplifier` is in fact an `lsst.afw.cameraGeom.Amplifier.Builder` object, and is passed as part of a list to `cameraGeom.addDetectorBuilderFromConfig`, which expects a list of `Amplifier.Builder`s.  This constructs the appropriate `Detector.Builder` attached to the `Camera.Builder`.  When `Camera.Builder.finish()` is called (at the end of `__new__`), all of the cameraGeom elements are frozen and converted to the non-builder versions as part of the final `Camera` object.